### PR TITLE
Trimmomatic-0.32

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/967b7340fec11e3d4a94 for a full test report.

File: easybuild/easyconfigs/t/Trimmomatic/Trimmomatic-0.32-Java-1.7.0_21.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ paired-end and single ended data.The selection of trimming steps and their associated 
+ parameters are supplied on the command line. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/']
+sources = [SOURCE_ZIP]
+
+java = 'Java'
+javaver = '1.7.0_21'
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/t/Trimmomatic/Trimmomatic-0.32-Java-1.7.0_21.eb]: LGTM; only a hint, not a blocker: java 1.7.0_80 is a bit more preferred (if, for common deps)

Comment (boegel) [inline easybuild/easyconfigs/t/Trimmomatic/Trimmomatic-0.32-Java-1.7.0_21.eb]: +1, let's bump this dep and merge @pes

**Comment by pescobar** [comment]
replaced by this one https://github.com/hpcugent/easybuild-easyconfigs/pull/1987

closing

**Pull request closed without merging**